src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-rg-lock/main.combined.bicep

```diff
@@ -0,0 +1,12 @@
+// $1 = resourceGroupLock
+// $2 = 'name'
+// $3 = 'NotSpecified'
+
+resource resourceGroupLock 'Microsoft.Authorization/locks@2017-04-01' = {
```

Comment:
Please could you use the latest API version here (`2020-05-01`)?

Reply:
I tried to use 2020-05-01, but it's not supported in Azure for some reason. See this error message: 
<img width="505" alt="image" src="https://user-images.githubusercontent.com/3514513/173077028-6355d1cf-d9c0-49e1-a94d-553749abb922.png">

Reply:
Thanks for checking! Let's leave as is then!